Rakefile

```diff
@@ -12,14 +14,28 @@ def collect_task_errors(tasks)
   errors.count.positive? && raise("The following tasks failed #{errors}")
 end
 
+def base_cucumber_path
+  "artifacts/#{ENV.fetch('BROWSER', 'unknown')}/#{ENV.fetch('BROWSERSTACK_CONFIGURATION_OPTIONS', 'other')}"
```

Comment:
Come to think of it this will still fail unless we delete our local configs. We have to have a conditional here. But we can still keep it split in 2.

```
if ENV["BROWSERSTACK"] == "true"
  "artifacts/#{ENV['BROWSER']}/#{ENV['BROWSERSTACK_CONFIGURATION_OPTIONS']}"
else
  "artifacts/#{ENV['BROWSER']}/other"
end
```

Reply:
Done